x/ibc/04-channel/keeper/keeper.go

```diff
@@ -139,6 +144,17 @@ func (k Keeper) deletePacketCommitment(ctx sdk.Context, portID, channelID string
 	store.Delete(host.KeyPacketCommitment(portID, channelID, sequence))
 }
 
+// deletePacketCommitmentsLTE removes all consecutive packet commitments less than or equal to sequence
+func (k Keeper) deletePacketCommitmentsLTE(ctx sdk.Context, portID, channelID string, sequence uint64) {
+	store := ctx.KVStore(k.storeKey)
```

Comment:
nit, this should use a prefix store. Idk if it's the case for the entire module but we should implement it as outlined here #5254